<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetBolaTipo</class>
 <widget class="QWidget" name="WidgetBolaTipo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nombreBola">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonParar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Para Bola</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombreBola_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Vidas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombreBola_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vidasBola">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cambiar el tipo de la bola</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCanbiarTipo1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCanbiarTipo2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCanbiarTipo3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombreBola_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TIpo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipoBola">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btEliminar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btRenombrar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Renombrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Renombrar la bola</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombreBola_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Colisiones:</string>
   </property>
  </widget>
  <widget class="QLabel" name="colisionesBola">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spVidas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btvidas">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Asignar vida</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
